### PR TITLE
My First TGIF!

### PR DESCRIPTION
**Pull request by skeskali**
If you have feedback, please share it. It's how I get better!

### CODE REVIEW

**Comment by janl** [comment]
OMG YES <333

I ran this locally and it looks all fine to me.

Thank you so much! Until now, I didn’t realise how much I missed TGIF!

File: _posts/2016-03-04-terrier-open-source-sensitive-democracy-for-hamilton-tles.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+[Mosaic Tiles That Resemble African Fabrics](http://design-milk.com/mosaic-tiles-resemble-african-fabrics/)
+
+[How the Heck Do You Hire A Web Design Agency?](https://css-tricks.com/how-the-heck-do-you-hire-a-web-design-agency/)
+
+## The Web and Development
+
+[Preload: What Is It Good For?](https://www.smashingmagazine.com/2016/02/preload-what-is-it-good-for/)
+
+## Events
+If you're in Washington, DC, come [Meet The Hoodies](https://ti.to/hoodie/dc-meet-the-hoodies) on Sunday, March 6.
```
Comment (gr2m) [inline _posts/2016-03-04-terrier-open-source-sensitive-democracy-for-hamilton-tles.md]: Can we add [Intro to Open Source](http://www.meetup.com/Jr-Dev-Mentoring/events/228951389/),  April 2, Atlanta. I’ll co-host with [Kim](https://twitter.com/KimCrayton1/)

**Comment by gr2m** [comment]
One minor: can we add `-tgif-73.md` to the filename? I’ve seen the previous ones are doing it, too

File: _posts/2016-03-04-terrier-open-source-sensitive-democracy-for-hamilton-tles.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+---
+title: "TERRIER OPEN SOURCE SENSITIVE DEMOCRACY FOR HAMILTON TILES:  TGIF! (73)"
```
Comment (gr2m) [inline _posts/2016-03-04-terrier-open-source-sensitive-democracy-for-hamilton-tles.md:2]: no need to do all caps, I think it’s done with CSS on the website later. See past TGIF for comparison https://raw.githubusercontent.com/hoodiehq/hood.ie/gh-pages/_posts/2015-07-10-golden-robots-tgif-72.md

**Comment by gr2m** [comment]
I’ve send a suggestion for Shout outs and updates on Hoodie Camp Release here: https://github.com/skeskali/hood.ie/pull/2/files 

Hope that works ^^?

**Pull request merged**